opentech/apply/funds/edit_handlers.py

```diff
@@ -0,0 +1,101 @@
+from django.forms.utils import pretty_name
+from django.urls import reverse
+from django.utils.html import format_html, mark_safe
+from django.utils.translation import ugettext_lazy as _
+
+from wagtail.wagtailcore.models import Page
+from wagtail.wagtailadmin.edit_handlers import BaseFieldPanel, EditHandler, FieldPanel
+
+
+class BaseReadOnlyPanel(EditHandler):
```

Comment:
I have a feeling the read-only panels may come handy further down the line. My feeling is they should go to separate app

Reply:
They will if needed again